quiche/src/recovery/gcongestion/bbr/bandwidth_sampler.rs

```diff
@@ -789,7 +791,7 @@ impl BandwidthSampler {
 
         while let Some(candidate) = a0_candidates.get(1) {
             if candidate.total_bytes_acked > total_bytes_acked {
-                return Some(*candidate);
+                break;
```

Comment:
Note that we were returning the `i` th element, instead of `i-1`

https://github.com/google/quiche/blob/66dc426/quiche/quic/core/congestion_control/bandwidth_sampler.cc#L524